### PR TITLE
MDA Chimera correction

### PR DESCRIPTION
**Pull request by ericsong**
Most of the code here is for the MDA chimera correction soft clipping procedure. I've also added the code to run Pacasus for we have no plans to use it in our pipelines since the performance is not up to what we are looking for.

### CODE REVIEW

**Comment by SHuang-Broad** [comment]
@ericsong Hey Eric, I took a quick look at the organization of the code.
Is it possible for you to move the python scripts under `scripts` to `docker/lr-softclipper/`?
We try to make all scripts under `scripts` as utilities for managing the workflows (validation, monitor, tests, etc) but not providing functions in the workflow tasks.

**Comment by ericsong** [comment]
Makes sense! Done.

File: wdl/tasks/Pacasus.wdl [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+    output {
+        File processed_fasta = MergeFasta.processed_fasta
+    }
+}
+
+task SplitReadsByLength {
+    input {
+        File reads_fasta
+        Int length
```
Comment (kvg) [inline wdl/tasks/Pacasus.wdl:50]: To be consistent with our method for overriding task runtime specs, please add the `RuntimeAttr? runtime_attr_override` to all task inputs.

File: wdl/tasks/Pacasus.wdl [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        seqkit seq ~{reads_fasta} --max-len $(expr ~{length} + 1) > shorter_reads.fasta
+        seqkit seq ~{reads_fasta} --min-len ~{length} > longer_reads.fasta
+    >>>
+
+    output {
+        File shorter_reads = "shorter_reads.fasta"
+        File longer_reads = "longer_reads.fasta"
+    }
+
+    runtime {
```
Comment (kvg) [inline wdl/tasks/Pacasus.wdl:67]: Coupled with the change above, the runtime section should have the RuntimeAttr preamble, like so:

```
    #########################
    RuntimeAttr default_attr = object {
        cpu_cores:          1,
        mem_gb:             1,
        disk_gb:            50,
        boot_disk_gb:       10,
        preemptible_tries:  3,
        max_retries:        1,
        docker:             "us.gcr.io/broad-dsp-lrma/lr-pb:0.1.8"
    }
    RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
    runtime {
        cpu:                    select_first([runtime_attr.cpu_cores,         default_attr.cpu_cores])
        memory:                 select_first([runtime_attr.mem_gb,            default_attr.mem_gb]) + " GiB"
        disks: "local-disk " +  select_first([runtime_attr.disk_gb,           default_attr.disk_gb]) + " HDD"
        bootDiskSizeGb:         select_first([runtime_attr.boot_disk_gb,      default_attr.boot_disk_gb])
        preemptible:            select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
        maxRetries:             select_first([runtime_attr.max_retries,       default_attr.max_retries])
        docker:                 select_first([runtime_attr.docker,            default_attr.docker])
    }
```

File: wdl/tasks/SoftClipper.wdl [hunk 3]
```diff
@@ -0,0 +1,103 @@
+version 1.0
+
+task SplitSoftClippedReads {
```
Comment (kvg) [inline wdl/tasks/SoftClipper.wdl:3]: Same comments as above regarding RuntimeAttr and runtime preamble.

**Review by kvg — APPROVED** [review]
Very nice work!  I'm gonna merge this PR and the other malaria PR.  I'll make a few very minor tweaks to the code in a separate PR.

**Pull request merged**